### ISSUE TITLE
Vessels disapear for no reason

### ISSUE THREAD

**Issue by ZaosDaOne** [darkevilmac/archimedes-ships-plus#241]
i have no clear lag plugin but ships randomly disappear for 0 reason
- 6.340
- 1.12.2
- 14.23.4.2732

**Comment by ZaosDaOne** [comment]
10/10 dont respond

**Comment by ghost** [comment]
yep sound like that is the way moders to day don't like to listen to people unless to do what they want

**Comment by ghost** [comment]
found a fix rase your render distance
